lib/uwb/protocols/fira/FiraDevice.cxx

```diff
@@ -202,14 +202,14 @@ std::string
 UwbRangingMeasurement::ToString() const
 {
     std::ostringstream ss{};
-    ss << "SlotIndex: " << +SlotIndex << ", "
-       << "Distance: " << Distance << ", "
-       << "Status: " << ::ToString(Status) << ", "
-       << "Peer Mac Address: " << PeerMacAddress << ", "
-       << "Line Of Sight Indicator: " << magic_enum::enum_name(LineOfSightIndicator) << ", "
-       << "Angle of Arrival Azimuth: " << AoAAzimuth << ", "
-       << "Angle of Arrival Elevation: " << AoAElevation << ", "
-       << "Angle of Arrival Destination Azimuth: " << AoaDestinationAzimuth << ", "
+    ss << "SlotIndex: " << +SlotIndex << "\n"
```

Comment:
For future consideration, we might have a "compact" string representation that packs all of the data in a single line with acronyms, something like:

```
Id: 287454020 Peer: bb:bb SEQ: 0 LoS D:10 AZ:-180 (100) E: -180 (100) AD: 180 (100) ED: 90 (100)
```

Then compact/normal/detailed formatting can be configured as a nocli command line option. I documented this in #304.